crates/rust-analyzer/src/tracing/config.rs

```diff
@@ -49,6 +52,10 @@ where
         let writer = self.writer;
 
         let ra_fmt_layer = tracing_subscriber::fmt::layer()
+            .with_timer(
+                time::OffsetTime::local_rfc_3339()
```

Comment:
sorry, to emphasize: i don't think this is enough. you're going to need to set https://docs.rs/time/latest/time/util/local_offset/fn.set_soundness.html if you want to use the `time` crate.

Reply:
@davidbarsky I don't understand, could you clarify?

This logic is invoked right at the start of r-a startup, and it's single threaded at this point. The default value of soundness is Sound anyway: https://docs.rs/time/latest/time/util/local_offset/enum.Soundness.html#variant.Sound

What am I missing / what's wrong with the code as written?

Reply:
I'm sorry: I mixed up `OffsetTime` with `LocalTime`. If [SystemTime](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/time/struct.SystemTime.html) doesn't work, `OffsetTime` is a good replacement.

Reply:
I've just tested SystemTime and it seems to use UTC rather than the local timezone unforunately. I think this is the best we can do.